### PR TITLE
tuc: add page

### PR DESCRIPTION
**Pull request by GregPK**
The examples are taken from original repo (https://github.com/riquito/tuc), just a shorter list.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/tuc.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# tuc
+
+> Cut text (or bytes) where a delimiter matches, then keep the desired parts.
+> A more user-friendly and powerful version of cut with sensible defaults.
+> More information: <https://github.com/riquito/tuc>.
+
+- Cut and rearrange fields:
+
+`\n echo "foo bar baz" | tuc -d ' ' -f 3,2,1    # => bazbarfoo`
```
Comment (gutjuri) [inline pages/common/tuc.md:9]:
```suggestion
`echo "foo bar baz" | tuc -d '{{ }}' -f {{3,2,1}}`
```

Why the `\n`?

File: pages/common/tuc.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Replace the delimiter with something else:
+
+`echo "foo bar baz" | tuc -d ' ' -r ' ➡ '  # => foo ➡ bar ➡ baz`
```
Comment (gutjuri) [inline pages/common/tuc.md:13]:
```suggestion
- Replace the delimiter `space` with an arrow:

`echo "foo bar baz" | tuc -d ' ' -r ' ➡ '`
```

File: pages/common/tuc.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+- Keep a range of fields:
+
+`echo "foo bar    baz" | tuc -d ' ' -f 2:   # => bar    baz`
```
Comment (gutjuri) [inline pages/common/tuc.md:17]:
```suggestion
- Keep a range of fields:

`echo "foo bar    baz" | tuc -d ' ' -f {{2:}}`
```

File: pages/common/tuc.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Cut using regular expressions:
+
+`echo "a,b, c" | tuc -e '[, ]+' -f 1,3  # => ac`
```
Comment (gutjuri) [inline pages/common/tuc.md:21]:
```suggestion
- Cut using regular expressions:

`echo "a,b, c" | tuc -e '{{[, ]+}}' -f {{1,3}}`
```

File: pages/common/tuc.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+- Emit JSON output:
+
+`echo "foo bar baz" | tuc -d ' ' --json     # => ["foo","bar","baz"]`
```
Comment (gutjuri) [inline pages/common/tuc.md:25]:
```suggestion
- Emit JSON output:

`echo "foo bar baz" | tuc -d '{{ }}' --json`
```

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks! I removed the shell comments that show the commands' outputs -- I don't thinks that we do this in other pages. Apart from that, I added placeholders.

File: pages/common/tuc.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
+# tuc
+
+> Cut text (or bytes) where a delimiter matches, then keep the desired parts.
+> A more user-friendly and powerful version of cut with sensible defaults.
```
Comment (spageektti) [inline pages/common/tuc.md:4]:
```suggestion
> A more user-friendly and powerful version of `cut` with sensible defaults.
```

**Comment by spageektti** [comment]
Hi. Thanks for your contribution. I wish a wonderful christmas time!

**Review by spageektti — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Comment by GregPK** [comment]
@sebastiaanspeck Ready. Needed a rebase.

**Pull request merged**